src-ocaml/baijiu_md5.ml

```diff
@@ -202,27 +202,26 @@ module Make (B : Baijiu_buffer.S)
 
     ()
 
-  let feed_bytes = feed ~blit:B.blit_from_bytes ~le32_to_cpu:B.le32_from_bytes_to_cpu
-  let feed_bigstring = feed ~blit:B.blit_from_bigstring ~le32_to_cpu:B.le32_from_bigstring_to_cpu
-  let feed = feed ~blit:B.blit ~le32_to_cpu:B.le32_to_cpu
+  let unsafe_feed_bytes = feed ~blit:By.blit ~le32_to_cpu:By.le32_to_cpu
+  let unsafe_feed_bigstring = feed ~blit:By.blit_from_bigstring ~le32_to_cpu:Bi.le32_to_cpu
 
-  let get ctx =
-    let padding = B.create 64 in
-    let bits = B.create 8 in
-    let res = B.create (4 * 4) in
+  let unsafe_get ctx =
+    let padding = By.create 64 in
+    let bits = By.create 8 in
```

Comment:
I prefer uninitialized memory to be declared very closely to where it is made safe again (by overwriting it), so I would move these down to where they are used.